configuration.rst

```diff
@@ -674,6 +674,53 @@ To define the value of an env var, you have several options:
 * :ref:`Encrypt the value as a secret <configuration-secrets>`;
 * Set the value as a real environment variable in your shell or your web server.
 
+It is possible to define an env var default value by defining a parameter with
```

Comment:
WDYT to explain in which case is needed ?

The only case is see is when env var if it's not set. If we don't configure this default symfony will throw an exception, right ?

Reply:
Isn't the "if it has not been defined anywhere" part on line 679 explicit enough? If no, I can add something else indeed!